C-Plus-Plus/ds/inverse of an array.cpp

```diff
@@ -0,0 +1,73 @@
+/*

```

Comment:
Name your file as `inverse_of_array.cpp`

Reply:
do i need to make a different PR with another file name. please help out. I dont know how to do that